automation/build.sh

```diff
@@ -69,6 +69,7 @@ cd "$COLLECTION_DIR"
 ansible-test sanity
 antsibull-changelog lint -v
 ansible-lint roles/* --exclude roles/hosted_engine_setup --exclude roles/disaster_recovery --exclude roles/remove_stale_lun -x experimental
+ansible-lint -x var-spacing -x ignore-errors -x var-naming roles/hosted_engine_setup
```

Comment:
Shouldn't `--exclude roles/disaster_recovery --exclude roles/remove_stale_lun` stay there as they should be fixed in https://github.com/oVirt/ovirt-ansible-collection/pull/543 ?

Reply:
I didn't change the line (71), only added a new one. I want specific exclusions for HE (as long as we need them). The new line only runs on HE, not on roles/* .

Reply:
I'm okay with skipping `var-naming` these changes could break backward compatibility.
`ignore-errors` please check out the article about `ignore_error` vs `changed_when` [1] but I'm okay with skipping this one if you have a reason that you want red ignored errors.
and `var-spacing` was already discussed 
[1] https://medium.com/@sbarnea/why-ansible-ignore-errors-is-evil-500fb6e81229

Reply:
I now read it, thanks for the pointer.

1. I do not completely disagree, just claim that going through each and every use of ignore_errors and spending the time investigating the exact reasoning, deciding on the best approach, etc., takes a lot of time, and I do not think it's mandatory to do for upgrading lint.

2. Quickly replacing every use with failed_when, IMO, is useless. At most, it makes the output slightly "nicer".

3. It's also not working, see above. Didn't investigate carefully. It seems like 'failed_when' can't be done for a block, where 'ignore_errors' apparently can be.

4. Most importantly: The reasoning of [1] applied also when we _added_ these ignore_errors. We should have been convinced at the time that they are ok. I now looked again at a few of them, and still think they make sense. Take e.g. "Check if keycloak is configured". The result is registered in keycloak_configured_out, which is used immediately later. So if the command failed in a way that will make the next task fail too, we'd notice, without further confusion, despite the use of ignore_errors. Similarly for "Wait for the host to be up". Some we really simply do not care about, e.g. "Generate the error message from the engine events" - this one is ran only if we already failed, and is only for outputting more information to the user. If this fails, ok - the user will have to spend some more time checking logs. No further harm here.

Reply:
This discussion is relevant for me as well :) in my [#543](https://github.com/oVirt/ovirt-ansible-collection/pull/543).
So is there a bottom line decision?
Should I skip `ignore_errors` or replace them with `failed_when`?

Reply:
yeah, I'll add `.ansible-lint` file for default ignores